.github/scripts/label_utils.py

```diff
@@ -77,12 +77,21 @@ def gh_get_labels(org: str, repo: str) -> List[str]:
 def gh_add_labels(
     org: str, repo: str, pr_num: int, labels: Union[str, List[str]]
 ) -> None:
-    gh_fetch_json(
-        f"https://api.github.com/repos/{org}/{repo}/issues/{pr_num}/labels",
+    gh_fetch_url(
+        url=f"https://api.github.com/repos/{org}/{repo}/issues/{pr_num}/labels",
+        headers={"Accept": "application/vnd.github.v3+json"},
```

Comment:
Again, not sure why this is needed (i.e. only important if one is going to parse the output)
```suggestion
```